consensus/state_processing/src/verify_operation.rs

```diff
@@ -39,12 +39,13 @@ pub trait TransformPersist {
 ///
 /// The inner `op` field is private, meaning instances of this type can only be constructed
 /// by calling `validate`.
-#[derive(Derivative, Debug, Clone)]
+#[derive(Derivative, Debug, Clone, Arbitrary)]
```

Comment:
I added this arbitrary implementation because my first reproduction used `cargo-fuzz`. It was quicker than wiring up `TestRandom`.

We don't usually check in `cargo fuzz` targets, but I figured there's no harm in leaving this in case we want to wire it up again in future.